documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -117,9 +117,29 @@ You can add source providers and destination providers for a virtual machine mig
 [id="adding-source-providers"]
 ==== Adding source providers
 
-You can add a VMware source provider, {a-rhv} source provider, or an {osp} source provider by using the {ocp} web console.
+You can use {project-short} to migrate VMs from the following source providers:
+
+* Red Hat {virt}
+* Red Hat {osp}
+* {rhv-full}
+* VMware vSphere
+* Open Virtual Appliances (OVAs) that originatate from or are compatible with VMware vSphere. 
```

Comment:
@ahadas The order is according what the user sees in the UI. I think it would be much better to change the layout of the UI than to ave a mismatch between the documentation and what te user sees.

Reply:
@yaacov Please let me know when you change the order of the icons. Thanks.

Reply:
@yaacov Will the order of the icons be changed before @.5 GA?